lib/x11utils.pm

```diff
@@ -105,9 +105,16 @@ sub ensure_unlocked_desktop {
             }, 10);
             record_info('Guest disabled display', 'Might be consequence of bsc#1168979');
         }
-        if (match_has_tag('authentication-required-user-settings') || match_has_tag('authentication-required-modify-system')) {
-            type_password;
-            assert_and_click "authenticate";
+      BSC1168979: {
```

Comment:
Please avoid using labels

Reply:
Because they alter the flow of the code, and makes it a bit more difficult to read for the untrained eye and we can end up in kind of spaghetti code (more than now), and this ensure_unlocked_desktop is already messy enough.

Reply:
Fair enough, though in this case I don't really see how not to make an minimum of spaghetti code.